set.mm

```diff
@@ -161144,6 +161193,15 @@ reflection about the origin (under the map ` x |-> -u x ` ).  (Contributed
     ( cc wcel cre cfv cabs recl cr recnd abscl syl cle wbr leabs absrele letrd
     ) ABCZADEZRFEZAFEAGZQRBCSHCQRTIRJKAJQRHCRSLMTRNKAOP $.
 
+  $( The square of the imaginary component of the square root of a complex
+     number is a non-negative real number.  (Contributed by RP,
```

Comment:
Wrong comment, probably a copy-paste error ?

Reply:
But Im[Sqrt[z]]^2 = (Abs[z] - Re[z])/2 holds for all z in the complex plane.

Reply:
I'm not saying the sentence is false.  I'm saying that
> The square of the imaginary component of the square root of a complex number is a non-negative real number.

cannot be the comment for the statement $A \in \mathbb{C} \to 0 \leq \frac{|A|- Re(A)}{2}$.

To exagerate things a bit, it's as if I commented the statement $\sin \pi = 0$ with the sentence "The limit when $x \to +\infty$ of $\sin(2\arctan x)$ is 0."

After looking at the statement, maybe the comment "Lemma for ~ sqrtcval ." is enough, given the elementary character of the statement (that is: no need IMO to paraphrase it).

Reply:
Further comment: what you just wrote above is your statement sqrtcval (and its Re and Im corollaries), and now I agree that these statements have some interest.